selenium_tests/course_info/page_objects/course_people_page_object.py

```diff
@@ -67,11 +77,38 @@ def select_role_type(self, canvas_role):
         self.find_element(*Locators.ROLES_DROPDOWN_LIST).click()
         self.find_element(By.LINK_TEXT, canvas_role).click()
 
-    def add_was_successful(self):
+    def single_add_was_successful(self):
         # Verify success text
-        # todo: this does not check _which_ add was successful...
+        # todo: we can refactor this; see "add_is_successful_by_id" method
+        try:
+            self.find_element(*Locators.SINGLE_ADD_ALERT_TEXT)
+        except NoSuchElementException:
+            return False
+        return True
+
+
+    def add_is_successful_by_id(self, user_id):
+        # Verify user ID is displayed
+        try:
+            self.find_element(*CourseInfoBasePageObjectLocators.TD_TEXT_XPATH(
```

Comment:
Successful adds are not displayed in the results table, if that's what we're attempting to match here. They _are_ displayed in the course members list, but may not be on the current page, depending on who's in the course and what the page size and ordering is, etc.